source/code-generation/query-converter/set-query-converter-llm.txt

```diff
@@ -0,0 +1,278 @@
+.. _rm-query-converter-set-llm:
+
+============================================================
+Set the Query Converter AI Provider and Large Language Model
+============================================================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+:ref:`Query Converter <rm-query-converter>` generates application code to
+streamline migrating an application from its original database
+into MongoDB. Query Converter is powered by generative AI. By default, it uses
+the GPT-4o model, hosted on MongoDB's `Azure OpenAI
+<https://azure.microsoft.com/en-us/products/ai-services/openai-service>`__
+subscription. You can configure Query Converter to use a different AI provider
+or Large Language Model (LLM) by following these steps.
+
+
+Before You Begin
+----------------
+
+- Review the list of :ref:`supported providers and models <rm-supported-llms>`.
+- Configure your AI provider.
+  This can include configuring access to the service, creating an API key or
+  access credentials for Query Converter, or provisioning other resources.
+  See your AI provider's documentation for details.
+
+Steps
+-----
+
+.. procedure::
+   :style: normal
+
+   .. step:: Stop Relational Migrator.
```

Comment:
Consider adding link or command to stop the migrator.

Reply:
Clarified this to: "Stop the Relational Migrator executable or service," but this is platform and deployment dependent, so linking to all variants would add a lot of bloat for little value.